### PR TITLE
Users can assign a property to a region

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

* Premises editing and creating pages now have a probation region drop-down menu

### Before
localhost_3000_properties_997cffc1-4414-424b-a4bb-1d39ae8c8e59 (1)
localhost_3000_properties_997cffc1-4414-424b-a4bb-1d39ae8c8e59_edit (2)

### After
localhost_3000_properties_997cffc1-4414-424b-a4bb-1d39ae8c8e59

localhost_3000_properties_997cffc1-4414-424b-a4bb-1d39ae8c8e59_edit (1)
localhost_3000_properties_997cffc1-4414-424b-a4bb-1d39ae8c8e59_edit

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Users can assign a property to a region](https://trello.com/c/1zgC2bZq/569-users-can-assign-a-property-to-a-region)

File: cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts [hunk 1]
```diff
@@ -2,18 +2,6 @@ import type { NewRoom, UpdateRoom } from '@approved-premises/api'
 import Page from '../../page'
 
 export default abstract class BedspaceEditablePage extends Page {
-  getCharacteristicIdByLabel(label: string, alias: string): void {
```
Comment (pezholio) [inline cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts]: I'm struggling to see what's going on in this commit - the json files that have been introduced don't seem to be used anywhere. Has something gone awry with the commit history?

Comment (bryangaledxw) [reply cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts]: Don't think so? There's no new JSON - the changed files are still used in `factories/referenceData.ts`

Where the JSON is now in sync with the API, a room or premises generated by our factories will by default be valid from the API's point-of-view. We don't have to do the extra step of reading valid characteristic/local authority IDs from the page in our Cypress tets, as the IDs we get from the factory will be fine

Does that make sense?

Comment (pezholio) [reply cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts]: Hmmmm - I'm not sure I follow still. Could you push this refactoring to a seperate branch for me to review? Might help me see the wood for the trees a bit.

Comment (tahb) [reply cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts]: In an effort to move this along i've read over the latest changes and it looks good and self contained now. It looks like this change was removed and perhaps the conversation continued on Slack?

I've run it locally and it works 👍 

@pezholio are you happy this comment is resolved?

Comment (pezholio) [reply cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts]: Yep, sorry!

**Review by pezholio — APPROVED** [review]
LGTM 👍

**Pull request merged**